pytest_inmanta/plugin.py

```diff
@@ -612,6 +623,41 @@ def _register_plugins(cls, project: module.Project) -> None:
             ):
                 plugins.PluginMeta.add_function(plugin)
 
+    @classmethod
+    def _unregister_old(cls, project: module.Project):
```

Comment:
Slight preference for the following name (`old` implies the existence of a `..._new` method to me). And if we move this to core, the signature will probably also include the modules as argument and return value.
```suggestion
    def _unregister_modules(cls, project: module.Project):
```